samples/nrf9160/serial_lte_modem/src/slm_at_tcpip.c

```diff
@@ -1028,8 +1070,17 @@ static int handle_at_send(enum at_cmd_type cmd_type)
 		if (err) {
 			return err;
 		}
-		data[size] = '\0';
-		err = do_send(data);
+		if (m_data_type == DATATYPE_HEXDECIMAL) {
+			u8_t data_hex[NET_IPV4_MTU/2];
```

Comment:
Spaces around operators, here and below: https://www.kernel.org/doc/html/latest/process/coding-style.html#spaces

Reply:
Updated